contrib/cirrus/setup_environment.sh

```diff
@@ -99,11 +99,29 @@ esac
 if ((CONTAINER==0)); then  # Not yet running inside a container
     # Discovered reemergence of BFQ scheduler bug in kernel 5.8.12-200
     # which causes a kernel panic when system is under heavy I/O load.
-    # Previously discovered in F32beta and confirmed fixed. It's been
-    # observed in F31 kernels as well.  Deploy workaround for all VMs
-    # to ensure a more stable I/O scheduler (elevator).
-    echo "mq-deadline" > /sys/block/sda/queue/scheduler
-    warn "I/O scheduler: $(cat /sys/block/sda/queue/scheduler)"
+    # Disable the I/O scheduler (a.k.a. elevator) for all environments,
+    # leaving optimization up to underlying storage infrastructure.
+    testfs="/"  # mountpoint that experiences the most I/O during testing
+    msg "Querying block device owning partition hosting the '$testfs' filesystem"
+    # btrfs filesystems append their internal label to the source name
+    # e.g. /dev/nvmeblah1p1[root] - filter this out
+    testdev=$(findmnt --canonicalize --noheadings --output source \
+              --mountpoint $testfs | cut -d '[' -f 1)
+    msg "    found partition: '$testdev'"
+    testdisk=$(lsblk -noheadings --output pkname --paths $testdev)
```

Comment:
Need two dashes in `--noheadings`. And please add a test for null output: `pkname` on my f35 system yields no result.

Reply:
Oops.  Got it, thanks.  Ya I'll check that and `testdev` as well, thanks for the suggestion.